templates/layout/page.html.twig

```diff
@@ -88,19 +84,27 @@
 
 <main class="main" id="main-content"> {# The "skip to content" link jumps to here. #}
 
+  {% if has_page_title and not node %}
```

Comment:
We'll need to put the Page Title block in the Page title region for this to work. 

https://github.com/localgovdrupal/localgov_microsites_base/blob/1.x/config/install/block.block.localgov_microsites_base_page_title.yml